properties/blender_object.py

```diff
@@ -13,10 +13,13 @@
     "Note that the random IDs of LuxCore can be greater than 32767 "
     "(the ID Mask node in the compositor can't handle those numbers)"
 )
-
+DESC_EXCLUDE_FROM_RENDER = ("The object will be excluded from render. "
+    "Useful if you need objects to render for other engines, but not for LuxCore."
+)
 
 class LuxCoreObjectProps(PropertyGroup):
     visible_to_camera: BoolProperty(name="Visible to Camera", default=True, description=DESC_VISIBLE_TO_CAM)
+    exclude_from_render: BoolProperty(name="Exclude from render", default=False, description=DESC_EXCLUDE_FROM_RENDER)
```

Comment:
This should be capitalized according to the convoluted rules of the english language: "Exclude from Render"